include/hip/hcc_detail/hip_runtime_api.h

```diff
@@ -41,6 +41,7 @@ THE SOFTWARE.
 
 #include <hip/hcc_detail/host_defines.h>
 #include <hip/hcc_detail/driver_types.h>
+#include <hip/hcc_detail/library_types.h>
```

Comment:
I don't see why it should be included here. The libraries that require it should include library_types.h similar to how it is in CUDA.